src/unittests/floating_point.spec.ts

```diff
@@ -5342,10 +5342,9 @@ const kMixImpreciseIntervalCases = {
       { input: [-1.0, 1.0, 0.9], expected: [reinterpretU64AsF64(0x3fe9_9999_8000_0000n), reinterpretU64AsF64(0x3fe9_9999_c000_0000n)] },  // ~0.8
 
       // Showing how precise and imprecise versions diff
-      // Note that this expectation is 0 only in f32 as 10.0 is much smaller that f32.negative.min,
+      // Note that this expectation is 0 in f32 as 10.0 is much smaller that f32.negative.min,
       // So that 10 - f32.negative.min == f32.negative.min even in f64. But for f16, there is not
-      // a exactly-represenatble f16 value v that make v - f16.negative.min == f16.negative.min
-      // in f64, in fact that require v being smaller than 2**-37.
+      // an exactly-represenatble f16 value v that make v - f16.negative.min == f16.negative.min
       { input: [kValue.f32.negative.min, 10.0, 1.0], expected: 0.0 },
```

Comment:
Would be nice to see some cases with `1.0` changed to something more than and less than `1.0`

Reply:
Done